### PR TITLE
Add node SSH settings serializable and routes

### PR DESCRIPTION
**Pull request by benbp**
Supports https://github.com/RackHD/on-core/pull/116

@RackHD/corecommitters @VulpesArtificem @heckj 

EDIT: whoops, forgot to add https://github.com/RackHD/on-core/pull/116 is required first.

### CODE REVIEW

File: spec/lib/api/2.0/nodes-spec.js [hunk 1]
```diff
@@ -359,6 +359,125 @@ describe('Http.Api.Nodes', function () {
...
+        var serializedSshSettings = {
+            host: '1.2.3.4',
+            user: 'myuser',
+            password: 'REDACTED'
+        };
+
+        it('should return a list of the node\'s ssh settings', function () {
+            waterline.nodes.needByIdentifier.resolves(sshNode);
+
+            return helper.request().get('/api/1.1/nodes/1234/ssh')
```
Comment (zyoung51) [inline spec/lib/api/2.0/nodes-spec.js]: Should these tests be targetting /api/2.0 instead of /api/1.1?

Comment (benbp) [inline spec/lib/api/2.0/nodes-spec.js]: good catch

**Comment by benbp** [comment]
Bear with me on tests here, having some rebase snafus.

**Comment by benbp** [comment]
Looks like there have been some updates to how we do views, I'll refactor in favor of that.

**Comment by benbp** [comment]
Closing this for now, rebase is screwing things up. I'll re-open when it's working.

**Comment by VulpesArtificem** [comment]
👍

**Comment by benbp** [comment]
test this please

**Pull request merged**